sale_margin_delivered/tests/test_sale_margin_delivered.py

```diff
@@ -60,10 +60,9 @@ def _new_sale_order(self):
     def test_sale_margin_ordered(self):
         sale_order = self._new_sale_order()
         order_line = sale_order.order_line[:1]
-        self.assertEqual(order_line.margin_delivered, 60.0)
-        self.assertEqual(order_line.margin_delivered_percent, 50.0)
-        self.assertEqual(
-            order_line.purchase_price_delivery, order_line.purchase_price)
+        self.assertEqual(order_line.margin_delivered, 0.0)
```

Comment:
So the test results were forced to fit the result? :sweat_smile:

Reply:
it's a change in behavior.